test/case/ietf_system/upgrade/test.py

```diff
@@ -33,17 +61,32 @@
             os.unlink(PKGPATH)
         except FileNotFoundError:
             pass
+        #os.unlink(PKGPATH)
         os.symlink(os.path.abspath(env.args.package), PKGPATH)
 
-        target = env.attach("target", "mgmt")
+        target = env.attach("target", "mgmt", "netconf")
+        target_ssh = env.attach("target", "mgmt", "ssh")
+        if target_ssh.run("test -e /sys/firmware/devicetree/base/chosen/u-boot,version".split()).returncode == 0:
+            bootloader=Uboot(target_ssh)
+        elif target_ssh.run("test -e /mnt/aux/grub/grubenv".split()).returncode == 0:
+            bootloader=Grub(target_ssh)
+        else:
+            print("No supported bootloader found")
+            test.skip()
+
+        old_bootorder=bootloader.get_boot_order()
```

Comment:
```suggestion
        old_bootorder=bootloader.get_boot_order()
        print(f"Initial bootorder: {repr(old_bootorder)}")
```
Since this referenced in many steps, I think logging it can help debugging future issues.